TMS.Net07.Homework4.RecursionAndFibonacci/TMS.Net07.Homework4.Task4.Fibonacci/TMS.Net07.Homework4.Task4.Fibonacci/Program.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System;
+
+namespace TMS.Net07.Homework4.Task4.Fibonacci
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            while (true)
+            {
+                Console.Write("Input a number to calculate a factorial: ");
+                long inputNumber = 0;
+                bool isTryInputNumber = false;
+
+                do
+                {
+                    isTryInputNumber = long.TryParse(Console.ReadLine(), out inputNumber);
+
+                    if (isTryInputNumber == false)
+                    {
+                        Console.Write("Incorrect input. Repeat, please: ");
+                    }
+                }
+                while (isTryInputNumber == false);
+
+                Console.WriteLine($"Result: {CalculateFibonacci(inputNumber)}");
+
+                Console.WriteLine();
+                Console.Write("Input anyting to continue or input \"exit\" for exit the program: ");
+                string isRepeatCycle = Console.ReadLine().ToLower();
+
+                if (isRepeatCycle == "exit")
+                {
+                    return;
+                }
+
+                Console.Clear();
+            }
+        }
+
+        static long CalculateFibonacci(long number)
+        {
+            if (number == 0 || number == 1)
```

Comment:
```suggestion
            if (number == 0 || number == 1 || number == -1)
```

Reply:
If I add a suggested condition then output for number -1 will be incorrect, because input -1 is equal output 1.
Tehre are fibonacci with simple recursion